app/src/test/java/com/x8bit/bitwarden/ui/platform/feature/settings/accountsecurity/AccountSecurityViewModelTest.kt

```diff
@@ -689,8 +706,26 @@ class AccountSecurityViewModelTest : BaseViewModelTest() {
         )
     }
 
+    @Suppress("MaxLineLength")
+    @Test
+    fun `when featureFlagManger returns true for AuthenticatorSync, and version is under 31 should not show authenticator sync UI`() {
+        every { isBuildVersionBelow(Build.VERSION_CODES.S) } returns true
+        val vm = createViewModel(
+            initialState = null,
+            featureFlagManager = mockk {
```

Comment:
Instead of replacing the whole thing, can we just mock what we need on the existing one?

```kotlin
every { featureFlagManager.getFeatureFlag(FlagKey.AuthenticatorSync) } returns true
every { featureFlagManager.getFeatureFlagFlow(FlagKey.AuthenticatorSync) } returns emptyFlow()
val vm = createViewModel(initialState = null)
```

Reply:
Cool! yes! updated!